tests/admin_utils/test_logentry.py

```diff
@@ -147,6 +147,15 @@ def test_logentry_get_edited_object(self):
         edited_obj = logentry.get_edited_object()
         self.assertEqual(logentry.object_id, str(edited_obj.pk))
 
+    def test_logentry_repr(self):
+        logentry = LogEntry.objects.first()
+        self.assertEqual(repr(logentry), force_text(logentry.action_time))
```

Comment:
Probably the `force_text()` will be replaced with `str()` as part of ticket #27795, similarly `str()` should give the same result here.